atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTestTools.java

```diff
@@ -64,6 +76,71 @@ public static void assertThatFutureDidNotSucceedYet(Future future) throws Interr
         }
     }
 
+    public static CqlResult setLocksTableValue(
+            CassandraClientPool clientPool,
+            UniqueSchemaMutationLockTable lockTable,
+            String hexLockValue,
+            ConsistencyLevel consistency) throws TException {
+        return clientPool.run(client -> {
+            String updateCql = "UPDATE \"%s\" SET value = %s WHERE key = %s AND column1 = %s AND column2 = -1;";
+            String lockRowName = getHexEncodedBytes(CassandraConstants.GLOBAL_DDL_LOCK_ROW_NAME);
+            String lockColName = getHexEncodedBytes(CassandraConstants.GLOBAL_DDL_LOCK_COLUMN_NAME);
+            String lockTableName = lockTable.getOnlyTable().getQualifiedName();
+            updateCql = String.format(updateCql, lockTableName, hexLockValue, lockRowName, lockColName);
+            return runCqlQuery(updateCql, client, consistency);
+        });
+    }
+
+    public static CqlResult truncateLocksTable(CassandraClientPool clientPool, UniqueSchemaMutationLockTable lockTable)
```

Comment:
This seems like a great time to move all the locks stuff into a `TestLockTable` object which takes in the `clientPool` and `lockTable` (maybe it should just take in the cassandra KVS instead)